mon-pix/app/components/authentication/login-or-register-oidc.hbs

```diff
@@ -2,39 +2,46 @@
 <div class="login-or-register-oidc-form__container">
   <div class="login-or-register-oidc-form__register-container">
     <h2 class="login-or-register-oidc-form__subtitle">{{t "pages.login-or-register-oidc.register-form.title"}}</h2>
-    <div>
-      <p class="login-or-register-oidc-form__description">
-        {{! template-lint-disable "no-bare-strings" }}
-        {{t "pages.login-or-register-oidc.register-form.description"}}
-        <em>{{this.identityProviderOrganizationName}}</em>&nbsp;:
-      </p>
-      <div class="login-or-register-oidc-form__information">
-        <ul>
-          <li>{{t "pages.login-or-register-oidc.register-form.information.given-name" givenName=this.givenName}}</li>
-          <li>{{t "pages.login-or-register-oidc.register-form.information.family-name" familyName=this.familyName}}</li>
-        </ul>
+    {{#if this.userClaims.length}}
```

Comment:
`this.userClaims` est un Object et pas un tableau. Donc le code pour faire cela serait plutôt `Object.keys(this.userClaims).length`.

Reply:
Dans le getter du controller, chaque userClaims est push dans un tableau incluant la trad et la valeur :)